pom.xml

```diff
@@ -134,8 +141,8 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.mockito</groupId>
```

Comment:
The project contains tests that use Mockito. How comes that they work when removing this dependency?

Reply:
interesting, I didn't do this intentionally, the tests still succeed however...

Reply:
Could you run a ```mvn dependency:tree``` to check who includes this lib? If it is not this ```pom.xml``` or a parent ```pom.xml``` than it is probably included via another dependency. in that case we should reintroduce the test dependency here.

Reply:
inherited via spring-boot-starter-test